mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -303,6 +346,8 @@ CLASS aij net/minecraft/entity/Entity
 		ARG 2 value
 	METHOD b isInvulnerableTo (Lahs;)Z
 		ARG 1 damageSource
+	METHOD b (Laij;)Z
+		ARG 1 e
```

Comment:
e? does the method not have a name either?

Reply:
Wait what? I think that's inside a lambda.

The `e` is a lambda-esque parameter name.

Reply:
ah, ok didnt know it was a lambada

Reply:
Maybe Enigma doesn't yet handle lambdas correctly?

Reply:
Lambdas are handled correctly, it is just confusing to read in this form

Reply:
Ah, okay. Well you might see a few other instances of that where I changed parameter names within lambdas.